### ISSUE TITLE
It would be awesome to remove the useless Xbox Game Bar.

### ISSUE THREAD

**Issue by frederikhors** [ChrisTitusTech/win10script#84]
It would be awesome to remove the useless Xbox Game Bar.

https://gist.github.com/joshschmelzle/04c57d957c5bb92e85ae9180021b26dc

**Comment by ChrisTitusTech** [comment]
I agree with you, but it will break a LOT of dependencies with Windows Store and Windows Store based Games.
